9.2/run-postgresql.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+
+# Data dir
+export PGDATA=/var/lib/pgsql/data
+
+# Enable utf8 support. This might affect performance:
+# http://www.postgresql.org/docs/9.2/static/locale.html
+export LANG=en_US.utf8
```

Comment:
this needs to be tuneable via env var... why is default not enough? can't you set this in postgres config file instead?